charts/orchestrator/templates/sonata-network-policy.yaml

```diff
@@ -0,0 +1,34 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: RHDH-to-sonata-runtimes
+spec:
+  podSelector:
+    matchExpressions:
+      - { key: sonataflow.org/workflow-app, operator: Exists }
```

Comment:
Are those labels already supported by the operator?

Reply:
I think it is https://redhat-internal.slack.com/archives/C059YMMB664/p1717597053038599